confluent_kafka/src/confluent_kafka.c

```diff
@@ -1123,6 +1157,17 @@ rd_kafka_conf_t *common_conf_setup (rd_kafka_type_t ktype,
 			}
 			Py_DECREF(ks);
 			continue;
+		} else if (!strcmp(k, "stats_cb")) {
```

Comment:
We might want to check here and for error_cb if the object is callable (`PyCallable_Check(vo)`) (Py_None is okay too)